.travis.yml

```diff
@@ -60,7 +62,7 @@ jobs:
     - stage: lint
       name: Lint (PHP, JavaScript, and configuration files)
       php: "7.4"
-      env: WP_VERSION=latest DEV_LIB_ONLY=xmllint,phpsyntax
+      env: WP_VERSION=latest DEV_LIB_ONLY=xmllint,phpsyntax,composer
```

Comment:
Adding the `composer` runner ensures that the local Composer binaries are used instead of the global ones installed by wp-dev-lib.